### PR TITLE
Use v1alpha2 as version in tests

### PR DESCRIPTION
**Pull request by vincepri**
Signed-off-by: Vince Prignano <vincepri@vmware.com>

**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

_Please confirm that if this PR changes any image versions, then that's the sole change this PR makes._

**Release note**:

```release-note

```

### CODE REVIEW

File: vendor/modules.txt [hunk 1]
```diff
@@ -364,7 +364,6 @@ k8s.io/apimachinery/pkg/runtime/serializer
 k8s.io/apimachinery/pkg/util/runtime
 k8s.io/apimachinery/pkg/watch
 k8s.io/apimachinery/pkg/apis/meta/v1/unstructured
-k8s.io/apimachinery/pkg/util/json
```
Comment (ncdc) [inline vendor/modules.txt]: Was this missed in some other PR?

**Comment by ncdc** [comment]
There's a lot of other code in here using v1alpha1 for infra & bootstrap.

File: vendor/modules.txt (hunk 1 above)
Comment (vincepri) [reply vendor/modules.txt]: I think so, #1174

**Comment by vincepri** [comment]
@ncdc I think I've got all the v1alpha2 related code changed, all the other matches seem to be in the deprecated package

**Pull request merged**